modules/client-common/test/context.test.ts

```diff
@@ -246,4 +246,31 @@ describe("Context instances", () => {
       LIVE_CONTRACTS.matic.ensRegistry,
     );
   });
+  it("Should create a context with baseGoerli as network and have the correct values", () => {
```

Comment:
Is there a way how we can do a bit more probing to make sure that some standard methods work with Base?
Exhaustive tests are certainly nothing for this PR, but having some probing tests could be helpful before you do E2E testing for the entire codebase.

Reply:
I created another test for testing that a client with the base context can be ccreated. I believe there isn't much we can do rn without doing a proper e2e test.